testkit/src/main/scala/org/bitcoins/testkit/node/NodeUnitTest.scala

```diff
@@ -274,10 +275,9 @@ object NodeUnitTest extends P2PLogger {
     for {
       bitcoind <- BitcoinSFixture.createBitcoindWithFunds(versionOpt)
       node <- createNeutrinoNode(bitcoind, callbacks)
-      chain <- node.chainApiFromDb()
       fundedWallet <- BitcoinSWalletTest.fundedWalletAndBitcoind(bitcoind,
                                                                  node,
-                                                                 chain)
+                                                                 node)
```

Comment:
same params here?

Reply:
Yes

Reply:
Yes

Reply:
Can you please start using [named arguments](https://docs.scala-lang.org/tour/named-arguments.html) in Scala so this is easier to code review, that would make this look like  

```scala
      fundedWallet <- BitcoinSWalletTest.fundedWalletAndBitcoind(bitcoindRpcClient = bitcoind,
                                                                 nodeApi = node,
                                                                 chainQueryApi = node)
```

which makes more sense to me! You can use if you click inside of the parameter list and press `alt+enter` in intellij they will automatically pop up for you.